$OpenBSD: patch-cmake_compilerFlags_cmake,v 1.5 2021/03/05 14:34:58 kmos Exp $

-fcf-protection isn't supported on all platforms (sparc64 in particular)

Index: cmake/compilerFlags.cmake
--- cmake/compilerFlags.cmake.orig
+++ cmake/compilerFlags.cmake
@@ -26,7 +26,7 @@ if ( MINGW OR UNIX OR MSYS ) # MINGW, Linux, APPLE, CY
         # This fails under Fedora, MinGW GCC 8.3.0 and CYGWIN/MSYS 9.3.0
         if (NOT (MINGW OR CMAKE_HOST_SOLARIS OR CYGWIN OR MSYS) )
             if (COMPILER_IS_GCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 8.0)
-                add_compile_options(-fstack-clash-protection -fcf-protection)
+                add_compile_options(-fstack-clash-protection)
             endif()
 
             if( (COMPILER_IS_GCC   AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 5.0) # Not in GCC 4.8
@@ -49,7 +49,7 @@ if ( MINGW OR UNIX OR MSYS ) # MINGW, Linux, APPLE, CY
             set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} --coverage")
         endif()
 
-        add_compile_options(-Wall -Wcast-align -Wpointer-arith -Wformat-security -Wmissing-format-attribute -Woverloaded-virtual -W)
+        add_compile_options(-Wall -Wcast-align -Wpointer-arith -Wformat-security -Woverloaded-virtual -W)
 
         # This seems to be causing issues in the Fedora_MinGW GitLab job
         #add_compile_options(-fasynchronous-unwind-tables)
